examples/with-auth/server.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+const session = require('express-session')
+const bodyParser = require('body-parser')
+const next = require('next')
+const auth = require('./auth')
+
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dir: '.', dev })
+const handle = app.getRequestHandler()
+
+app.prepare()
+.then(() => {
+  const server = express()
+
+  server.use(bodyParser.json())
+  server.use(session({ secret: 'Meow', resave: false, saveUninitialized: true }))
+  server.use(auth.sessionSupport())
+  server.use(auth.acceptToken({ successRedirect: '/' }))
+
+  server.post('/sendtoken', auth.requestToken((email, delivery, fn) =>
+  fn(null, email)), (req, res) => res.json('ok'))
+
+  server.get('/me', (req, res) => res.json(req.user || null))
+
+  server.post('/logout', (req, res) => req.session.destroy(() => res.json('ok')))
```

Comment:
Maybe use Auth0 (as I do so far in my proto) then people can start from there if they want to use their own stuff?